app/assets/stylesheets/views/_homepage.scss

```diff
@@ -251,3 +251,14 @@
     border-top: 4px solid $govuk-brand-colour;
   }
 }
+
+// Temporary overrides for global bar styling on homepage during election.
+// Doing this here avoids the need for republishing the gem and therefore
+// will be easier to remove later.
+.homepage.global-bar-present .global-bar {
+  border: 0;
+}
+
+.homepage.global-bar-present .homepage-header {
+  padding-top: 55px;
```

Comment:
This looks quite big on mobile - is that intentional?

Reply:
We've added 40px here as the previous padding was 15px, but you're right no mobile design was provided. I'll double check if it needs tweaking for mobile.

Reply:
This has now been updated after pairing with Nikin and tweaking the values for different breakpoints.